config/manager/kustomization.yaml

```diff
@@ -9,3 +9,7 @@ images:
 - name: open-component-model/git-controller
   newName: ghcr.io/open-component-model/git-controller
   newTag: latest
+
+# Uncomment to enable HTTPS for the registry
+#patches:
+#- path: ./patches/add_root_certificates.yaml
```

Comment:
Should this be the other way around because HTTPS is enabled on the registry by default?
```suggestion

# Comment to disable HTTPS for the registry
patches:
- path: ./patches/add_root_certificates.yaml
```